src/renderer/src/layouts/default/index.tsx

```diff
@@ -13,7 +13,9 @@ export const DefaultLayout = ({
 }) => {
   return (
     <Box minHeight="100vh" minWidth="100vw" sx={sx}>
-      <LanguageSwitcher langBackgroundVarient={langBackgroundVarient} />
+      <div style={{ zIndex: 1 }}>
+        <LanguageSwitcher langBackgroundVarient={langBackgroundVarient} />
+      </div>
```

Comment:
added this because the tab panel content interferes with the dropdown (can't press it), probably due to the absolute positioning of the tab panel content